src/test/java/com/github/cameltooling/lsp/internal/diagnostic/CamelDiagnosticTest.java

```diff
@@ -42,7 +42,7 @@
 
 public class CamelDiagnosticTest extends AbstractCamelLanguageServerTest {
 
-	private static final Duration AWAIT_TIMEOUT = Duration.ofSeconds(10);
+	private static final Duration AWAIT_TIMEOUT = Duration.ofSeconds(100000);
```

Comment:
27.7 hours timeout?

Reply:
you're not ready to wait? :-D

I forgot to switch it back, good catch

Reply:
I already feared that Duration.ofSeconds might be a confusing method name.